custom_components/localtuya/pytuya/__init__.py

```diff
@@ -537,7 +543,7 @@ def _generate_payload(self, command, data=None):
             json_data["dps"] = self.dps_to_request
 
         payload = json.dumps(json_data).replace(" ", "").encode("utf-8")
-        self.log.debug("Paylod: %s", payload)
+        self.log.debug("Send paylod: %s", payload)
```

Comment:
Tested with my official environment and everything works. Just this little typo we might want to correct (paylod).

Reply:
Certainly! Fixed now.